linker/js/src/main/scala/org/scalajs/linker/NodeIRContainer.scala

```diff
@@ -23,15 +23,15 @@ import scala.scalajs.js.typedarray.TypedArrayBufferOps._
 
 import java.nio._
 
-import org.scalajs.linker.standard.{IRContainerImpl, IRFileImpl}
+import org.scalajs.linker.interface.{IRContainer, IRFile}
+import org.scalajs.linker.interface.unstable.IRContainerImpl
 import org.scalajs.linker.standard.MemIRFileImpl
 
-abstract class IRContainerPlatformExtensions private[linker] () {
+object NodeIRContainer {
```

Comment:
Shouldn't this refactoring be in its own commit, like the other refactorings in this PR?